backend/src/Designer/Services/Implementation/SchemaModelService.cs

```diff
@@ -85,10 +85,19 @@ public IList<AltinnCoreFile> GetSchemaFiles(string org, string repository, strin
         /// <inheritdoc/>
         public async Task<string> GetSchema(string org, string repository, string developer, string relativeFilePath, CancellationToken cancellationToken = default)
         {
-            cancellationToken.ThrowIfCancellationRequested();
-            var altinnAppGitRepository = _altinnGitRepositoryFactory.GetAltinnAppGitRepository(org, repository, developer);
+            try
+            {
+                cancellationToken.ThrowIfCancellationRequested();
+                var altinnAppGitRepository =
+                    _altinnGitRepositoryFactory.GetAltinnAppGitRepository(org, repository, developer);
+
+                return await altinnAppGitRepository.ReadTextByRelativePathAsync(relativeFilePath);
+            }
+            catch (FileNotFoundException)
+            {
+                return null;
```

Comment:
It's not a best practice to return null. It would be better for caller to handle specific exception